pkg/cmd/projectconfig/import.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package projectconfig
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/charmbracelet/huh"
+	apiclient_util "github.com/daytonaio/daytona/internal/util/apiclient"
+	"github.com/daytonaio/daytona/pkg/apiclient"
+	"github.com/daytonaio/daytona/pkg/views"
+	views_util "github.com/daytonaio/daytona/pkg/views/util"
+	"github.com/daytonaio/daytona/pkg/views/workspace/create"
+	"github.com/daytonaio/daytona/pkg/views/workspace/selection"
+	"github.com/spf13/cobra"
+)
+
+var projectConfigList []apiclient.ProjectConfig
+
+var projectConfigImportCmd = &cobra.Command{
+	Use:                   "import [-]",
```

Comment:
Apologies, I misspoke in my last comment.

The command shouldn't be `daytona pc import -`. Rather, it should be `daytona pc import -f -` where `-f` (`--file`) can accept a file from where to import the PC. If the file argument is set to `-`, then we import from stdin.

Reply:
Not a problem, you can check now!